src/test/kotlin/com/launchdarkly/intellij/toolwindow/FlagNodeBaseTest.kt

```diff
@@ -0,0 +1,20 @@
+package com.launchdarkly.intellij.toolwindow
+
+import org.junit.jupiter.api.Test
+
+import org.junit.jupiter.api.Assertions.*
+
+internal class FlagNodeBaseTest {
+    private val testBase: FlagNodeBase = FlagNodeBase(label = "Test")
```

Comment:
should we use a `setUp` functions for defining this?

Reply:
I followed the basic setup here: https://kotlinlang.org/docs/jvm-test-using-junit.html#create-a-test

I just quickly tried and it becomes unhappy with `testTargets` inside the `getChildren` test. I think we might be able to do a setup with a `lateinit var` but my preference would be to merge this in as is, then we can keep making tweaks to it.

Reply:
works for me